package.json

```diff
@@ -3,14 +3,23 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@fortawesome/fontawesome-svg-core": "^1.2.25",
+    "@fortawesome/free-solid-svg-icons": "^5.11.2",
+    "@fortawesome/react-fontawesome": "^0.1.6",
     "@material-ui/core": "^4.5.1",
     "axios": "^0.19.0",
+    "extends": "^0.0.4",
     "formik": "^1.5.8",
+    "less-watch-compiler": "^1.14.1",
+    "material-ui": "^0.20.2",
```

Comment:
We can delete this line. We are using `@material-ui/core`

 Side note, Material UI also comes with Icons you can use. 
https://material-ui.com/components/material-icons/

Reply:
great, just played with material UI icons.  works just as well, so I removed the font awesome libraries from package.json locally.